packages/database/package.json

```diff
@@ -22,8 +22,6 @@
   },
   "license": "Apache-2.0",
   "peerDependencies": {
```

Comment:
Should these become dev dependencies?

Reply:
`@firebase/app` can't be made a dependency because then it will be installed with admin sdk, which can cause potential version conflict if `firebase` is installed as well.

Reply:
Just realized you said dev dependencies - I don't feel strongly about it, but I think it's nice the dependencies are preserved this way (though not 100% correctly).